ignite3/src/main/java/site/ycsb/db/ignite3/IgniteSqlClient.java

```diff
@@ -72,7 +96,15 @@ public Status read(String table, String key, Set<String> fields, Map<String, Byt
   /** {@inheritDoc} */
   @Override
   public Status update(String table, String key, Map<String, ByteIterator> values) {
```

Comment:
An overrided method `IgniteTxSqlClient.update` still returns `Status.NOT_IMPLEMENTED`. Was it left unchanged intentionally or not?

Reply:
Same story with `IgniteTxJdbcClient.update` method.